apps/ledger-live-mobile/e2e/specs/nftGallery.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import { loadConfig } from "../bridge/server";
+import { tapByElement } from "../helpers";
+import WalletTabNavigatorPage from "../models/wallet/walletTabNavigator";
+import NftViewerPage from "../models/nft/nftViewerPage";
+import NftGalleryPage from "../models/wallet/nftGalleryPage";
+
+let portfolioPage: PortfolioPage;
+let walletTabNavigatorPage: WalletTabNavigatorPage;
+let nftGalleryPage: NftGalleryPage;
+let nftViewerPage: NftViewerPage;
+
+describe("NFT Gallery screen", () => {
+  beforeAll(async () => {
+    await loadConfig("2_accounts_eth_with_tokens_and_nft", true);
+    portfolioPage = new PortfolioPage();
```

Comment:
Can the pages go outside the `beforeAll` or do they need to do in here?

Reply:
I've noticed the old tests have them like that too. Will check if that's necessary

Reply:
Indeed i copied this, but i think it's not necessary at all

Reply:
Ok, let's leave for now and I can tidy it up later